res/mock_evm/src/lib.rs

```diff
@@ -1,6 +1,9 @@
 use crate::out::SubmitResult;
-use aurora_workspace_types::input::{CallInput, DeployErc20Input, NewInput};
 use aurora_workspace_types::output::{Log, TransactionStatus};
+use aurora_workspace_types::{
+    input::{CallInput, DeployErc20Input, NewInput},
+    Raw20,
```

Comment:
Please correct me if I am wrong why we have to use `RAW20` where we can use [H160](https://docs.rs/primitive-types/latest/primitive_types/struct.H160.html)

Reply:
Also type `Address` type is wrapping for H160 https://docs.rs/ethereum-types/latest/ethereum_types/type.Address.html

Reply:
Interesting, `Raw20` shouldn't be here anymore, it was removed in the previous PR. Git merge probably did the wrong thing when rebasing, there were no conflicts...

Reply:
Good question